### ISSUE TITLE
Teaching Tip that blocks interactions with app window like Content Dialog

### ISSUE THREAD

**Issue by nilaysanghvi** [microsoft/microsoft-ui-xaml#3267]
Is it possible to display a Teaching Tip that blocks UI interactions until explicitly dismissed? A content dialog does this where the user can only interact with the dialog or close the app -
 
![image](https://user-images.githubusercontent.com/70956797/92607141-5a94a880-f2d1-11ea-99e0-1415793f5b8e.png)

Is there any workaround through which we replicate the above functionality of the content dialog for teaching tip?

**Comment by ranjeshj** [comment]
@nilaysanghvi [TeachingTip](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/dialogs-and-flyouts/teaching-tip) is designed to not block user interactions with the app. Can you explain the scenario you are trying to achieve, perhaps you need to use a different control ?

**Comment by nilaysanghvi** [comment]
@ranjeshj My scenario is that on clicking a button I want to show 2-3 targetted Teaching tips to the user to highlight the core functionalities of the app (X button does A, Y pane contains B, etc). While showing these tips I do not want to give the option to the user to interact with the other elements in the app or exit this guided flow. Such guided tours are frequently seen in Android apps for which external [libraries](https://github.com/amlcurran/ShowcaseView) are available.

**Comment by StephenLPeters** [comment]
One of teaching tip's core design principles was to not be modal. However this does seem like legitimate teaching scenario teaching tip could service.  Could you open a feature proposal so we can think about including this in a future update.  As a work around you could probably disable your app while the tips are open using either the Disabled Property or the IsHitTestVisible property on your root element. (the teaching tips would be in the popup root) you might also want to open a smoke layer to not confuse your users.

**Comment by nilaysanghvi** [comment]
@StephenLPeters Yes sure, I will open a feature proposal. 
Could you please give some more idea about the smoke layer and how to do add that? It would be really helpful to me!

**Comment by nilaysanghvi** [comment]
@StephenLPeters Thanks for your suggestions, I was able to add the UI blocking behaviour that I needed. Whenever the tip is displayed, I initialise a user control - which is simply a grid with background taken from the [Content Dialog](https://github.com/microsoft/microsoft-ui-xaml/blob/dc08d9b429755dd73e581d45764d61a4205f94d4/dev/ContentDialog/ContentDialog_themeresources.xaml#L87). Then I add this user control to the root element of my app. This grid blocks all the clicks and also serves the purpose of the smoke layer. I also set the Teaching Tip's LightDismiss to False, so that on clicking on the grid the Teaching tip is not closed.
